### DISCUSSION TITLE
not working as advertised

### DISCUSSION THREAD

**Discussion by THS1994** [aclap-dev/video-downloadhelper#1506] [Help and Q&A]
### Your question

bought the premium liscence as it told me i needed one to download more then x amount of length 120m in a certain timeframe. 

Now even with it, it wont allow downloads or if it does the entire video file is distorted. Refund me my purchase

### Addon & CoApp version and details

bought the premium liscence as it told me i needed one to download more then x amount of length 120m in a certain timeframe. 

Now even with it, it wont allow downloads or if it does the entire video file is distorted. Refund me my purchase

**Comment by Mattmjs** [comment]
That isn't the fault of the software or the license, some sites use DRM to protect the media files.

**Comment by jcfrog** [comment]
Hi, if you need a refund please send us an email at licenses@downloadhelper.net

**Reply by THS1994** [reply]
sent one but waiting for a response. also is netflix blocked from this or not?

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

VDH is not blocked from any site, including Netflix.  What does prevent VDH from working is DRM.  If your content is protected by DRM, then VDH will not successfully download it.  DRM exists for the expressed purpose of preventing downloading of copyrighted material.  There are numerous discussions in this forum on the topic of DRM.  Here's a recent one: #1504.  That one shows how you can determine whether a particular video is protected by DRM.

In addition to DRM, we have encountered a number of other weird things that web sites do that interfere with the correct processing of VDH.  These range from protecting manifest files with undocumented passwords to wildly non-standard ways of presenting multimedia data.  So DRM is not the only reason VDH can fail.  You can try to use screen recording software in such cases.  You would be on your own in such a case.  I know screen recording is not preferable to downloading, but sometimes it's the best you can do.

As for support, you need to realize that @jcfrog Jérôme who, unlike me, is an employee of VDH, lives in France.  So you need to take the time zone difference into account to calibrate your expectations.  Not to mention the fact that it's a weekend.  You just need to be patient.  Or, you could always just resend your E-mail.

**Reply by THS1994** [reply]
yeah ive sent a email 2x now & it fails to send. and then im getting account errors for my email almost like im being hacked. im filing a chargeback

**Reply by GrampaWildWilly** [reply]
Try sending an E-mail to yourself.  That would be just a diagnostic to make sure it's working properly.  If that passes, then make sure you aren't mistyping Jérôme's E-mail address.  Double check it above.  A lot of people have communicated with him over the years.  He's not trying to hide.  That address does work.  You can't be the only person in the world unable to communicate with him.

**Reply by GrampaWildWilly** [reply]
You might also want to check with tech support at your ISP.  There may be a problem they can help you solve.

**Reply by THS1994** [reply]
had nothing to do with my email turns out. it was sending but their server wasnt accepting it. cause it would say sent & not a draft or couldnt send

**Reply by THS1994** [reply]
i cant be bothered to keep having my emails rejected. a chargeback will solve this issue. if he doesnt want to respond in this forum to fix it then thats what it will be. sure as shit isnt my isp. i can send a email just like normal. its this stupid address

**Reply by GrampaWildWilly** [reply]
Are you saying E-mails to licenses@downloadhelper.net are being rejected?  Not E-mails to GitHub.  E-mails to licenses@downloadhelper.net.  Are you sure you're spelling that correctly?  He says E-mail him because the information he needs is the sort of stuff that should not be posted in public.  It's not that he refuses to answer you here.  He posts here all the time.  It's that your particular case requires the exchange of non-public information.

**Reply by THS1994** [reply]
well it says ive sent 3 different emails to the address. whether he got it or not i dont know. ive never gotten a response

so im assuming hes not receiving it. and im copying it from this forum so yes its correct & its simply titled Refund

**Reply by THS1994** [reply]
like if this is continuing to be a issue he can contact me via like discord even. i have a private server. ill exchange information there

**Reply by GrampaWildWilly** [reply]
As a test, I just sent an E-mail to licenses@downloadhelper.net & it seems to have gone through.  But I have seen it take a number of hours before a mail server manages to return an error message so maybe I'm being too hasty.  Still, it's awfully fishy that you can't send an E-mail to that address.

**Reply by GrampaWildWilly** [reply]
Well good.  At least you've managed to get an E-mail to send without an error message from the server saying it's rejected your message.  Now all you can do is wait.  Based on history, Jérôme is very responsive.  I just hope nothing untoward is going on with him.

**Reply by jcfrog** [reply]
@THS1994 sorry your emails don't reach us. Please give me the 10 first digits of your license and I'll refund you.

**Reply by THS1994** [reply]
aa75f12e1

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days. (replied to your email too)

**Comment by THS1994** [comment]
Thank You.
Sent from my iPhone

On Dec 3, 2024, at 5:51 AM, jcfrog ***@***.***> wrote:

﻿

You have been refunded. Should be available in a few business days. (replied to your email too)
